lib/view_model/active_record/association_data.rb

```diff
@@ -196,6 +196,21 @@ def viewmodel_class
     viewmodel_classes.first
   end
 
+  def ordered?
```

Comment:
There are a few remaining places that use `._list_member?` instead of `ordered?`. It'd be nice to clean those up, even if there's no observable difference (`load_associated`, `append_associated`).

Reply:
Sure, done. There's still one remaining use of _list_member?, in a context that doesn't have the association data: https://github.com/iknow/iknow_view_models/blob/support-sti/lib/view_model/active_record/update_operation.rb#L652